graphite/graphite.go

```diff
@@ -79,6 +84,12 @@ func (f *GraphitePublisher) Publish(metrics []plugin.Metric, cfg plugin.Config)
 		}
 		data := fmt.Sprintf("%v", m.Data)
 		logger.Debug("Metric ready to send %s:%s", key, data)
+		for _, char := range notAllowedChars {
+			if strings.ContainsAny(key, char) && strings.ContainsAny(data, char) {
+				logger.Errorf("Illegal characters found. Error: %s", err)
+				return fmt.Errorf("Illegal characters: %s", err)
```

Comment:
Do we really want to error them out? Should we use urlEncoding? Let us experiment it a bit.